pkg/ui/workspaces/cluster-ui/src/util/docs.ts

```diff
@@ -61,6 +61,9 @@ export const statementsTimeInterval = docsURL(
 export const readFromDisk = docsURL(
   "architecture/life-of-a-distributed-transaction.html#reads-from-the-storage-layer",
 );
+export const writtenToDisk = docsURL(
```

Comment:
wondering if this is the right article to link, or if I should just keep the "readFromDisk" link?

Reply:
They both go to the same document, but the `#write-operations` subsection deals specifically with writes.